src/hooks/useSymptom.ts

```diff
@@ -0,0 +1,132 @@
+import { useNavigation } from "@react-navigation/native";
+import { useEffect, useState } from "react";
+import Voice, {SpeechResultsEvent,SpeechErrorEvent,SpeechRecognizedEvent,} from '@react-native-voice/voice';
+import { useMutation } from "@tanstack/react-query";
+import axios from "axios";
+export const useSymptom=()=>{
+  const [started, setStarted] = useState('');
```

Comment:
I think we can use a state called isRecording.
true mean is started
false means is end